lighthouse-cli/test/smokehouse/smokehouse-report.js

```diff
@@ -126,6 +126,11 @@ function makeComparison(name, actualResult, expectedResult) {
  * @return {Smokehouse.Comparison[]}
  */
 function collateResults(actual, expected) {
+  // If actual run had a runtimeError, expected *must* have a runtimeError.
+  // Relies on the fact that an `undefined` argument to makeComparison() can only match `undefined`.
+  const runtimeErrorAssertion = makeComparison('runtimeError', actual.lhr.runtimeError,
```

Comment:
doh! everything is off of `lhr` now we should just iterate through the keys

Reply:
> everything is off of lhr now we should just iterate through the keys

agreed, though I really want to make sure we check `runtimeError`, even if it's not in the expectations